tests/PHPUnit/Integration/DbSSLTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tests\Integration;
+
+use Piwik\Db;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
+use Piwik\Config;
+
+/**
+ * @group Core
+ * @requires PHP 5.3.7
+ */
+class DbSSLTest extends IntegrationTestCase
+{
+    public function testMysqlSSLConnection() {
+        $dbConfig = Config::getInstance()->database;
+        if(isset($dbConfig['enable_ssl']) AND $dbConfig['enable_ssl'] == true) {
+            Db::createDatabaseObject($dbConfig);
+            $cipher = Db::fetchRow("show status like 'Ssl_cipher'");
+            $this->assertNotEmpty($cipher['Value']);
+        } else {
+            $this->isTrue(true);
```

Comment:
either mark it true or mark it skipped with message like "enable_ssl = 0, test skipped"